6-compiler.js

```diff
@@ -5,11 +5,11 @@ import { transformer } from './4-transformer.js';
 import { codeGenerator } from './5-code-generator.js';
 
 const button = document.getElementById('button');
-const input = document.getElementById('input');
-const text = document.getElementById('text');
+const inputSyntax = document.getElementById('inputSyntax');
```

Comment:
Syntax isn't really the right word here. Syntax refers more to the structure of the text but you care about the meaning of the contents. You could probably just go with a simple and intuitive `input` & `output`. It's definitely better than `input` & `text`.

Reply:
🙆🏻‍♀️